src/element-transformer/index.ts

```diff
@@ -0,0 +1,132 @@
+import * as ts from 'typescript';
+
+export default function transformer(program: ts.Program): ts.TransformerFactory<ts.SourceFile> {
```

Comment:
I'm concerned about how this PR will integrate due to this parameter being passed. The test code sets up the compiler correctly but code that incorporates it probably needs to be adjusted.

Reply:
I would rather not have the setting up of this happen in the webpack config if possible.